sympy/core/relational.py

```diff
@@ -305,15 +305,17 @@ def _eval_simplify(self, **kwargs):
         r = self
         r = r.func(*[i.simplify(**kwargs) for i in r.args])
         if r.is_Relational:
+            if r.lhs.is_Set and r.rhs.is_Set:
+                if r.lhs.is_subset(r.rhs) and r.rhs.is_subset(r.lhs):
+                    return True
+                else:
+                    return False
             dif = r.lhs - r.rhs
```

Comment:
The `is_subset` function does not always return True or False. You need to consider the indeterminate case e.g.:
```
In [1]: Eq({1}, {x})
Out[1]: {1} = {x}
```

Reply:
Also the block below should only run if lhs and rhs are both Expr.